sig-scalability/extending_slo.md

```diff
@@ -0,0 +1,71 @@
+# Extended Kubernetes scalability SLOs
```

Comment:
Since we will be adding more SLOs, I suggest creating a directory for SLO-related documents.